app/javascript/components/record-form/form/subforms/subform-header/component.jsx

```diff
@@ -83,6 +83,7 @@ const Component = ({ field, values, locale, displayName, index, isViolationSubfo
 
     return (
       <ListItemText id="subform-header-button" classes={itemClasses}>
+        {values[index][`${field.name.split("_section")[0]}_date`]}
```

Comment:
This probably is not the solution for this issue. Can you confirm that your configuration has: `collapsed_field_names: ['felt_stigma_date']` and that after loading the configuration this becomes: collapsed_field_names: ['felt_stigma_scale']

Reply:
I have removed this code. I think someone has fixed the issue. Collapsed field is working for me.
![Screenshot 2022-03-17 14:42:05](https://user-images.githubusercontent.com/5393463/158781964-82f2f618-769e-4aa8-9093-eff47037f33d.png)